nautobot_floor_plan/utils.py

```diff
@@ -20,3 +23,13 @@ def grid_letter_to_number(letter):
     if letter[:-1]:
         return 26 * (grid_letter_to_number(letter[:-1])) + number
     return number
+
+
+def validate_not_zero(value):
```

Comment:
Not blocking, but to get around the validate_not_zero.message and follow established Django Pattern, you can change this to a class. In case you need this in the future.

```suggestion
from django.core.validators import BaseValidator

class NonZeroValidator(BaseValidator):
    """
    Ensure that the field's value is not zero.
    """
    message = "Must be a positive or negative Integer not equal to zero."
    code = "zero_not_allowed"
    
    def __call__(self, value):
        if value == 0:
            raise ValidationError(
                self.message,
                code=self.code,
            )
```